tools/development/helpers/format-python.sh

```
@@ -2,11 +2,9 @@
 
 # Apache License 2.0
 
-project_directory="$(git rev-parse --show-toplevel)"
+pushd "${PYTHON_WD}" > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)